x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -430,6 +487,36 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               data-test-subj="mlAnalyticsCreateJobFlyoutDestinationIndexInput"
             />
           </EuiFormRow>
+          <EuiFormRow
```

Comment:
Seeing as this field is optional, I wonder if it should appear below the dependent variable and training percent controls for regression and classifications jobs:

![image](https://user-images.githubusercontent.com/7405507/71676001-fb13ef80-2d76-11ea-83e8-490a42e44f2c.png)

Reply:
Good point - moved in 24bdeb3e4dbf7b711cb9f257118c356b6baa8c38